src/test/unit/marker-data.test.js

```diff
@@ -230,6 +231,22 @@ describe('deriveMarkersFromRawMarkerTable', function() {
       title: null,
     });
   });
+  it('should create a marker for the marker UserTiming', function() {
+    const { markers } = setup();
+    expect(markers[13]).toMatchObject({
+      data: {
+        type: 'CompositorScreenshot',
+        url: 16,
+        windowID: '0x136888400',
+        windowWidth: 1280,
+        windowHeight: 1000,
+      },
+      name: 'CompositorScreenshot',
+      start: 25,
+      dur: -18,
```

Comment:
Hmmm, btw I'm not sure why this is `-18` here.

Reply:
It's being adjusted here:

https://github.com/firefox-devtools/profiler/blob/master/src/profile-logic/marker-data.js#L371-L374